Jan 8 - Help center, nav, dev tools and overview, sharing, VCCs

rasienko opened the pull request:
@sthielen  For your review, here’s a draft of the first news update of the year. It includes the nav changes and new features you released over the holidays. Let me know if I missed/misunderstood anything or if you have anything else planned for release this week. I tentatively set the date as Friday in case there were some other things we wanted to include. Also, Tim will be updating the VCC screenshots to reflect the new styling, but I expect that will be ready soon. Thanks!

sthielen commented on src/asciidoc/docs/develop/testingTemplates.adoc:
```diff
@@ -230,6 +177,10 @@ You can view real-time events related to the sticker context.
 * *Dynamic Receipt* – Debug the receipts that are sent to the buyer and seller, when in-app purchases are implemented with dynamic receipts.
 You can view real-time events related to the receipts.
 
+TIP: To preview a custom context, you can override the environment settings to append the parameter to the URL.
```
I'm not sure I understand this - is this about showing different contexts inside the debugger? Is this not solved by changing the tabs in the top right?

rasienko replied:
Thanks! The previous version of the page talked about how to test a custom context (vs. a built-in e.g. sticker) in the live player. Is that still an available test mode, or should we just remove this tip if it's confusing?

sthielen reviewed (commented):
Thanks! This all makes sense to me aside from one piece that I found confusing, but I might not be understanding what it is.

sthielen commented:
I think we can remove it - all of the alternate contexts should be
available in the debugger tabs or the editor live preview buttons

On Wed, Jan 6, 2021 at 4:52 PM rasienko <notifications@github.com> wrote:

> *@rasienko* commented on this pull request.
> ------------------------------
>
> In src/asciidoc/docs/develop/testingTemplates.adoc
> <https://github.com/madewithkoji/docs/pull/118#discussion_r553048188>:
>
> > @@ -230,6 +177,10 @@ You can view real-time events related to the sticker context.
>
>  * *Dynamic Receipt* – Debug the receipts that are sent to the buyer and seller, when in-app purchases are implemented with dynamic receipts.
>
>  You can view real-time events related to the receipts.
>
>
>
> +TIP: To preview a custom context, you can override the environment settings to append the parameter to the URL.
>
>
> Thanks! The previous version of the page talked about how to test a custom
> context (vs. a built-in e.g. sticker) in the live player. Is that still an
> available test mode, or should we just remove this tip if it's confusing?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/madewithkoji/docs/pull/118#discussion_r553048188>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA5VPJZSFS6NYLWXWU22JJDSYUAVFANCNFSM4VYGXQBA>
> .
>

rasienko commented:
@sthielen I removed that confusing part and also updated some navigation instructions that I missed the first time around. Tim's changes are merged as well, so this PR is ready to go pending your approval. Let me know if we missed anything or need any additional changes. Thanks!

sthielen reviewed (approved):
Looks good to me!
